rules.ini

```diff
@@ -44,7 +44,16 @@ ChromeEngine[] = \.rpack$
 Clausewitz[] = (?:^|/)common/defines\.(?:lua|txt)$
 Clausewitz[] = ^(?:game/)?common/defines/00_defines\.txt$
 ClickTeamFusion = (?:^|/)mmfs2\.dll$
-Cocos2d = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)Cocos2d\.lua$
+Cocos[] = (?:^|/)jsb-adapter/jsb-engine\.jsc?$
+Cocos[] = (?:^|/)jsb-adapter/engine-adapter\.jsc?$
+Cocos[] = (?:^|/)cocos2d-jsb\.[0-9a-z]+\.jsc?$
+Cocos[] = (?:^|/)cocos2d-jsb\.jsc?$
+Cocos[] = (?:^|/)assets/main/cc\.config\.json$
+Cocos[] = (?:^|/)assets/main/cc\.config\.[0-9a-z]+\.json$
+Cocos[] = (?:^|/)cocos-js/cc\.jsc?$
+Cocos[] = (?:^|/)cocos-js/cc\.[0-9a-z]+\.jsc?$
```

Comment:
You can remove this one
only 1 match and it's the same depot as `(?:^|/)jsb-adapter/engine-adapter\.jsc?$`
* [2129201 - ](https://steamdb.info/depot/2129201/)
```suggestion
```